integrations/website-indexer/integration.definition.ts

```diff
@@ -0,0 +1,49 @@
+import { IntegrationDefinition, z } from '@botpress/sdk'
+import { integrationName } from './package.json'
+
+export default new IntegrationDefinition({
+  name: integrationName,
+  version: '0.0.1',
+  readme: 'hub.md',
+  icon: 'icon.svg',
+  secrets: {
+    SCRAPER_API_KEY: {
+      description: 'Scaper API Key',
+      optional: false,
+    },
+  },
+  actions: {
+    indexUrls: {
+      title: 'Index URLs',
+      description: 'Indexes URLs',
+      input: {
+        schema: z.object({
+          // studio does not support string[] input
+          pageUrls: z.string(),
+        }),
+      },
+      output: {
+        schema: z.object({
+          // studio does not support string[] output
+          fileIds: z.string(),
```

Comment:
:writing_hand: **todo:** zui was updated recently on the studio so we switched from the old form to the new, list of strings are supported so we don't need special handling